classes/class-wc-connect-help-provider.php

```diff
@@ -2,20 +2,263 @@
 
 if ( ! class_exists( 'WC_Connect_Help_Provider' ) ) {
 
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_WOOCOMMERCE_VERSION', '2.6' );
```

Comment:
Fixed (moved constants) in https://github.com/Automattic/woocommerce-connect-client/pull/349/commits/a7353cea96a89ad17253944326fb8c299a88c9b3